diff --git a/node_modules/open/index.js b/node_modules/open/index.js
index f66dc4c..6bc00c4 100644
--- a/node_modules/open/index.js
+++ b/node_modules/open/index.js
@@ -255,9 +255,11 @@ const open = (target, options) => {
 };
 
 const openApp = (name, options) => {
-	if (typeof name !== 'string') {
-		throw new TypeError('Expected a `name`');
-	}
+	console.log("DEBUG openApp called with", name, options)
+	// if (typeof name !== 'string') {
+	// 	console.error("DEBUG name type is", typeof name)
+	// 	throw new TypeError('Expected a `name`');
+	// }
 
 	const {arguments: appArguments = []} = options || {};
 	if (appArguments !== undefined && appArguments !== null && !Array.isArray(appArguments)) {
